src/admin-views/admin-welcome-message.php

```diff
@@ -2,84 +2,223 @@
+}
 ?>
-</p>
 
+<?php if ( $has_plus ) : ?>
+	<div class="tribe-events-admin-notice">
+		<div class="tribe-events-admin-content-wrapper">
+			<img
+				class="tribe-events-admin-notice__logo"
+				src="<?php echo esc_url( tribe_resource_url( 'images/icons/horns-white.svg', false, null, $main ) ); ?>"
+				alt="<?php esc_attr_e( 'The Events Calendar product suite logo', 'event-tickets' ); ?>"
+			/>
+			<p><strong><?php echo esc_html_x( 'WOOHOO!', 'short expression of excitement', 'event-tickets' ); ?></strong> <?php esc_html_e( 'You\'re the proud owner of Event Tickets Plus! Let\'s get started…', 'event-tickets' ); ?></p>
+		</div>
+	</div>
+<?php endif; ?>
 
-<div class="tribe-row">
+<div class="tribe-events-admin-content-wrapper tribe-events-admin-tickets <?php if ( $has_plus ) { echo 'tribe-events-admin-tickets-plus'; } ?>">
+	<img
+		class="tribe-events-admin-graphic tribe-events-admin-graphic--desktop-only"
+		src="<?php echo esc_url( tribe_resource_url( $desktop_graphic, false, null, $main ) ); ?>"
+		alt="<?php esc_attr_e( 'Shapes and lines for visual interest', 'event-tickets' ); ?>"
+	/>
 
-	<div class="tribe-half-column">
-		<h4 data-tribe-icon="dashicons-welcome-learn-more"><?php esc_html_e( 'Getting Started', 'event-tickets' ); ?></h4>
-		<p><?php esc_html_e( 'Start strong with these helpful resources.', 'event-tickets' ); ?></p>
-		<ul>
-			<li><a href="https://theeventscalendar.com/product/wordpress-event-tickets/" target="_blank"><?php esc_html_e( 'Key Features', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9u" target="_blank"><?php esc_html_e( 'Settings Overview', 'event-tickets' ); ?></a></li>
-			<li><a href="http://m.tri.be/1a9v" target="_blank"><?php esc_html_e( 'New User Primer', 'event-tickets' ); ?></a></li>
-		</ul>
+	<img
+		class="tribe-events-admin-graphic tribe-events-admin-graphic--mobile-only"
+		src="<?php echo esc_url( tribe_resource_url( $mobile_graphic, false, null, $main ) ); ?>"
+		alt="<?php esc_attr_e( 'Shapes and lines for visual interest', 'event-tickets' ); ?>"
+	/>
+
+	<div class="tribe-events-admin-title">
+		<img
+			class="tribe-events-admin-title__logo"
+			src="<?php echo esc_url( tribe_resource_url( $logo_image, false, null, $main ) ); ?>"
+			alt="<?php esc_attr_e( 'Event Tickets logo', 'event-tickets' ); ?>"
+		/>
+		<h2 class="tribe-events-admin-title__heading">
+			<?php
+			if ( $has_plus ) :
+				esc_html_e( 'Event Tickets Plus', 'event-tickets' );
+			else :
+				esc_html_e( 'Event Tickets', 'event-tickets' );
+			endif;
+			?>
+		</h2>
+		<p class="tribe-events-admin-title__description"><?php
+			if ( $has_plus ) :
+				esc_html_e( 'Thanks for installing Event Tickets Plus! Here are some handy resources for getting started with our plugins.', 'event-tickets' );
+			else :
+				esc_html_e( 'Thanks for installing Event Tickets! Here are some handy resources for getting started with our plugins.', 'event-tickets' );
+			endif;
+		?></p>
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected 3 tabs, found 2
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 3a856f56902c3acf87548bda36c171ad</sub>